sycl/plugins/level_zero/CMakeLists.txt

```diff
@@ -0,0 +1,81 @@
+# PI Level0 plugin library
+
+message(STATUS "Download Level Zero loader and headers from github.com")
```

Comment:
Can we have an option to provide local version of the L0 headers and libraries like it is done for OpenCl dependencies?

Reply:
I can do this. But I have a question about how it is done for OpenCL. In configure.py:
```
 21     ocl_header_dir = os.path.join(abs_obj_dir, "OpenCL-Headers")
 22     icd_loader_lib = os.path.join(abs_obj_dir, "OpenCL-ICD-Loader", "build")
```
These paths are used if we pass --system-ocl to configure. But they are not system paths. 
So  this option/behavior is a little confusing. Shouldn't we look system paths (like /usr/local/lib or smth) if such option is provided?

Reply:
Added ability to provide local version of headers and loader.